Swift/Tests/ReplicatorTest+Collection.swift

```diff
@@ -84,38 +84,38 @@ class ReplicatorTest_Collection: ReplicatorTest {
         
         var config = ReplicatorConfiguration(database: self.db, target: target)
         
-        let conflictResolver: TestConflictResolver = TestConflictResolver({ (con: Conflict) -> Document? in
+        let conflictResolver = TestConflictResolver({ (con: Conflict) -> Document? in
             return con.remoteDocument
         })
         config.conflictResolver = conflictResolver
         XCTAssertNotNil(conflictResolver)
         
         var colConfig = collectionConfig(config, col: try defaultCollection())
-        XCTAssertEqual(conflictResolver, (config.conflictResolver as! TestConflictResolver))
-        XCTAssertEqual(conflictResolver, (colConfig.conflictResolver as! TestConflictResolver))
+        XCTAssert(conflictResolver === (config.conflictResolver as! TestConflictResolver))
```

Comment:
I don't think `(config.conflictResolver as! TestConflictResolver)` is needed? Just `conflictResolver === config.conflictResolver`.

Reply:
type difference, config.conflictResolver is type ConflictResolverProtocol. Other one is class.